metronome/hotstuff/consensus/test/src/io/iohk/metronome/hotstuff/consensus/ArbitraryInstances.scala

```diff
@@ -8,13 +8,18 @@ import io.iohk.metronome.hotstuff.consensus.basic.Phase.{
 }
 import io.iohk.metronome.hotstuff.consensus.basic.VotingPhase
 import org.scalacheck.Arbitrary.arbitrary
+import org.scalacheck.rng.Seed
 import org.scalacheck.{Arbitrary, Gen}
 import scodec.bits.ByteVector
 
 trait ArbitraryInstances {
 
   def sample[T: Arbitrary]: T = arbitrary[T].sample.get
 
+  //TODO: rename / remove above?
```

Comment:
I thought I was going to use it in order to avoid creating extra fixture parameters to pass where `Gen` context is not available, but I did not in the end. Remove?

Reply:
I'm not exactly sure how to use `sample0`, where to get the `Seed` from, but we can leave it in for rainy days.